internal/mode/static/state/graph/route_common_test.go

```diff
@@ -1406,332 +1411,6 @@ func TestValidateHostnames(t *testing.T) {
 	}
 }
 
-func TestValidateFilterRequestHeaderModifier(t *testing.T) {
```

Comment:
Tests for filters that are common between GRPC and HTTP routes have been moved to `common_filter_test.go`